src/main/java/com/parsing/model/LaptopItem.java

```diff
@@ -0,0 +1,41 @@
+package com.parsing.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Entity
```

Comment:
It would be best if you had a repository for testing. I think you'll be able to create a repository for this task. And push it.

Reply:
Done!